EpiAware/test/predictive_checking/toy_model_log_infs_RW.jl

```diff
@@ -67,6 +67,9 @@ using StatsPlots
 using Random
 using DynamicPPL
```

Comment:
Is this intensionally missing things from the test environmment? Like StatsPlots?

Reply:
`StatsPlots` is in the test env but not a dependency of `EpiAware`; which I think is the right move.